repos/spack_repo/builtin/packages/emacs/package.py

```diff
@@ -63,6 +63,7 @@ class Emacs(AutotoolsPackage, GNUMirrorPackage):
     )
     variant("json", default=False, when="@27:", description="Build with json support")
     variant("native", default=False, when="@28:", description="Enable native compilation of elisp")
+    variant("sqlite", default=False, when="@29:", description="Build with sqlite3 support")
```

Comment:
```suggestion
    variant("sqlite", default=False, when="@29.1:", description="Build with sqlite3 support")
```

It seems like `sqlite-mode` is only available in v29.1 and newer.